packages/devtools_app/lib/src/performance/performance_controller.dart

```diff
@@ -85,6 +85,9 @@ class PerformanceController
       _httpTimelineLoggingEnabled;
   final _httpTimelineLoggingEnabled = ValueNotifier<bool>(false);
 
+  ValueListenable<bool> get badgeTabForJankyFrames => _badgeTabForJankyFrames;
+  final _badgeTabForJankyFrames = ValueNotifier<bool>(true);
```

Comment:
should we default this to true in profile builds and false in debug builds?

Reply:
done